public/stylesheets/webcamCSS.css

```diff
@@ -8,10 +8,19 @@
 /*    height: 100%; */
 /*}*/
 
+.alert {
```

Comment:
The alert will look strange when the error occurs because master currently only has the container fluid changes in two spots! I didn't fix the formatting since you will be making a PR and I didn't want to rely on the old design!

Reply:
Hmmm what do you mean? How come it looks weird?

Reply:
In the last PR I added in a few of the bootstrap changes to webcam but not everything so a few things are rendering weirdly on my end! I'll probably make a new PR in the future to fix the weird formatting if it happens after your webcam PR is merged

Reply:
Ohh I see okay makes sense